packages/wonder-blocks-modal/components/modal-launcher.js

```diff
@@ -103,15 +112,6 @@ export default class ModalLauncher extends React.Component<Props, State> {
     };
 
     static getDerivedStateFromProps(props: Props, state: State) {
-        if (typeof props.opened === "boolean" && props.children) {
-            throw new Error("'children' and 'opened' can't be used together");
-        }
-        if (typeof props.opened === "boolean" && !props.onClose) {
-            throw new Error("'onClose' should be used with 'opened'");
-        }
-        if (typeof props.opened !== "boolean" && !props.children) {
-            throw new Error("either 'children' or 'opened' must be set");
-        }
```

Comment:
Should we at least log a warning? Flow doesn't preclude having runtime checks for bad data.

Reply:
I've added warnings and tests for the warnings.  This is probably good practice for any constraints like this that depend on flow.